sections/05_treat.md

```diff
@@ -1,35 +1,42 @@
 ## The impact of deep learning in treating disease and developing new treatments
 
 `TODO: write Treat intro`
+
+`TODO: just realised this wanders heavily into the categorization area. Need some high-level rearrangment? Could shift the first section below into the categorization section and leave the treatment here.`
+
+Given the ever-present need to make better interventions faster at the point of care -- incorporating the complex calculus of a patients
+symptoms, diagnostics and life history -- there is a long history of attempts to
+apply deep learning to the problem of treatment. Success in this area would also be very useful for directions like personalised healthcare or precision medicine [@doi:10.1056/NEJMp1006304 @doi:10.1155/2013/769639]. Broadly, these may divided into
+methods for improved classification of patients (i.e. improved and those for development of
+new interventions.
+
 `TODO: Add representative papers about Modeling Metabolism and Chemical
 Reactivity`
 
 ### Categorizing patients for clinical decision making
 
-There has been sustained interest in applying deep learning to clinical
-decision making for over two decades. In 1996, Tu
+As long ago as 1996, Tu
 [@tag:Tu1996_anns] compared the effectiveness of artificial
 neural networks and logistic regression, questioning whether deep learning
 would replace traditional statistical methods for predicting medical
 outcomes such as myocardial infarction [@tag:Baxt1991_myocardial] or
 mortality [@tag:Wasson1985_clinical]. He posited that while neural
 networks have several advantages in representational power, the difficulties in
-interpretation may limit clinical applications. In 2006, Lisboa and Taktak
+interpretation may limit clinical applications. Similarly, in 2006 Lisboa and Taktak
 [@tag:Lisboa2006_review] examined the use of artificial neural
-networks in medical journals, concluding that neural networks provided an
-increase in benefit to healthcare relative to traditional screening methods in
+networks in medical journals, concluding that they improved healthcare relative to traditional screening methods in
 21 of 27 studies.
 
-While significant progress has been made in developing deep learning methods
-for diagnosis, it is not clear that these methods have yet transformed
-clinical decision making. The difficulty in applying deep learning to clinical
```

Comment:
It might be fine to cut this line, but "clinical decision making" is the intended division between this Treat section and the Categorize sections.  Categorize pertains to clinical data, but Treat was supposed to focus more on actionable predictions.